### PR TITLE
Remove miner from erigon command

### PR DESCRIPTION
**Pull request by boyuan-chen**
Remove miner command

### CODE REVIEW

File: ops-bedrock/l2-erigon.sh [hunk 1]
```diff
@@ -15,9 +14,6 @@ COMMON_FLAGS=" \
 
 ERIGON_FLAGS=" \
   ${COMMON_FLAGS} \
-  --mine \
```
Comment (jyellick) [inline ops-bedrock/l2-erigon.sh:18]: Removing the etherbase and the sigfile seems fine, but if `--mine` isn't enabled, then I don't believe erigon will build blocks and act as a sequencer.

I just tried removing it in `op-e2e/extrernal_erigon/main.go` and the tests simply hang for me.  If we're removing it here, I'd think we should remove it there as well?

Comment (boyuan-chen) [reply ops-bedrock/l2-erigon.sh:18]: I have removed it in our sequencer node. It's still building new blocks.

Comment (jyellick) [reply ops-bedrock/l2-erigon.sh:18]: Looks like this breaks the e2e in CI as well -- does it work on your system?

Comment (boyuan-chen) [reply ops-bedrock/l2-erigon.sh:18]: No, I am trying to figure out why. The integration tests work, but op-e2e tests don't.

Comment (boyuan-chen) [reply ops-bedrock/l2-erigon.sh:18]: Ok, figured out. The `Regeneration ended` observation should be removed. Geth also has this `--mine` option and it is not enabled in the op-e2e tests. I think we should be good to remove it.
https://github.com/ethereum-optimism/op-geth/blob/0402d543c3d0cff3a3d344c0f4f83809edb44f10/cmd/utils/flags.go#L457-L461

File: op-e2e/external_erigon/main.go [hunk 2]
```diff
@@ -160,10 +158,6 @@ func execute(binPath string, config external.Config) (*erigonSession, error) {
 	}
 	fmt.Fscanf(sess.Err, "%d", &enginePort)
 	fmt.Printf("==================    op-erigon shim got engine port %d  ==========================\n", enginePort)
-	gm.Eventually(sess.Err, time.Minute).Should(gbytes.Say("Regeneration ended"))
```
Comment (jyellick) [inline op-e2e/external_erigon/main.go:163]: I think we should probably look for another message that indicates startup is done -- and we can hopefully eliminate the sleep below.  But that's outside the scope of this PR, I'll take a look and submit separately.

**Review by jyellick — APPROVED** [review]

**Review by mmontour1306 — APPROVED** [review]

**Pull request merged**